ThunkLibs/include/common/Host.h

```diff
@@ -72,6 +72,7 @@ struct fex_guest_function_ptr {
 
 #define EXPORTS(name) \
   extern "C" { \
+    __attribute__((visibility("default"))) \
```

Comment:
Why is this needed? Aren't those functions only accessed through the exports table?

Reply:
Oops, that was when I was overreaching with default hidden visibility. Removed.